templates/publisher/account-snaps.html

```diff
@@ -170,3 +172,20 @@ <h4>My registered snap names</h4>
   {% endif %}
 </section>
 {% endblock %}
+
+{% block scripts %}
+<script>
+  Raven.context(function () {
+    var message = document.querySelector('.js-close-message');
+    if (message) {
+      message.addEventListener('click', function(e) {
+        e.preventDefault();
+        var parent = message.parentNode;
```

Comment:
We have a polyfill for `el.closest()` so may be better to use that instead of assuming it will be always 2 parents.

Reply:
Done, thanks @bartaz